stock/templates/registered/corputilssearchview.html

```diff
@@ -0,0 +1,57 @@
+{% extends "public/base.html" %}
+{% load bootstrap %}
+{% load staticfiles %}
+
+{% block title %}Alliance Auth{% endblock %}
+
+{% block page_title %}Corporation Member Tracking{% endblock page_title %}
+{% block extra_css %}{% endblock extra_css %}
+
+{% block content %}
+    <div class="col-lg-8">
```

Comment:
This needs to be wrapped in a `<div class="col-lg-12">` block to render properly - it's misaligned right now:

![search](http://i.imgur.com/GTMiDdU.png)